### PR TITLE
[DOCS] Add request body parameters to put Logstash pipeline API docs

### LINKED ISSUE #69148: [DOCS] Put Logstash pipeline API docs are missing request body parameter definitions
The put Logstash pipeline API docs are missing definitions for the request body parameters:

https://www.elastic.co/guide/en/elasticsearch/reference/master/logstash-api-put-pipeline.html

### PR DESCRIPTION
**Pull request by jrodewig**
Fixes #69148

Additional changes:

- Changes `pipeline` to lowercase
- Adds links to the Centralized Pipeline Management docs
- Adds title abbreviations

### Preview
Request body parameters: https://elasticsearch_69151.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/logstash-api-put-pipeline.html#logstash-api-put-pipeline-request-body

### CODE REVIEW

File: x-pack/docs/en/rest-api/logstash/put-pipeline.asciidoc [hunk 1]
```diff
@@ -19,25 +24,59 @@ This API creates or updates Pipelines used for Logstash Central Management.
...
+`pipeline_settings`::
+(Required, object)
+Settings for the pipeline. Supports only flat keys in dot notation. For
+supported settings, see the {logstash-ref}/logstash-settings-file.html[{ls}
+settings documentation].
+
+`username`::
+(Required, string)
+User who last updated the pipeline. This username is not generated or used by
+{es} or {ls}.
```
Comment (jrodewig) [inline x-pack/docs/en/rest-api/logstash/put-pipeline.asciidoc:68]: > This username is not generated or used by {es} or {ls}.
@jaymode Can you confirm this is the case? Thanks!

Comment (jaymode) [reply x-pack/docs/en/rest-api/logstash/put-pipeline.asciidoc:68]: As far as I know, this is the case. @kaisecheng do you know of any uses of the username field?

**Review by jaymode — APPROVED** [review]
LGTM

File: x-pack/docs/en/rest-api/logstash/put-pipeline.asciidoc (hunk 1 above)
Comment (kaisecheng) [reply x-pack/docs/en/rest-api/logstash/put-pipeline.asciidoc:68]: @jaymode the `username` display in the pipeline table of central pipeline management, so it is used by Logstash
https://github.com/elastic/kibana/blob/master/x-pack/plugins/logstash/public/application/components/pipeline_list/pipelines_table.js#L59

Comment (jrodewig) [reply x-pack/docs/en/rest-api/logstash/put-pipeline.asciidoc]: Thanks for confirming @kaisecheng. I removed the related sentence.

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/2 

@elasticmachine run elasticsearch-ci/bwc

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Pull request merged**